portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -4089,23 +4086,15 @@ public void unsetRoleUsers(long roleId, List<User> users)
 	public void unsetRoleUsers(long roleId, long[] userIds)
 		throws PortalException {
 
-		Role role = rolePersistence.findByPrimaryKey(roleId);
-
-		String roleName = role.getName();
-
-		if ((roleName.equals(RoleConstants.ADMINISTRATOR) &&
-			 (getRoleUsersCount(role.getRoleId()) <= 1))) {
+		List<User> users = new ArrayList<>(userIds.length);
 
-			throw new RequiredRoleException.RequiredAdminRoleException();
-		}
+		for (Long userId : userIds) {
+			User user = userLocalService.fetchUser(userId);
```

Comment:
hey @ChrisKian 

fetchMethod could return a null object, we should use a get (or do a null check)